src/components/indicators/indicators.js

```diff
@@ -15,37 +16,33 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
         return false;
     }
 
-    function getProgressHtml(pct, options) {
-        var containerClass = 'itemProgressBar';
-        if (options) {
-            if (options.containerClass) {
-                containerClass += ' ' + options.containerClass;
-            }
+    export function getProgressHtml(pct, options) {
+        let containerClass = 'itemProgressBar ';
+        if(options && options.containerClass) {
+            containerClass += options.containerClass
```

Comment:
I would leave `containerClass` without spaces, as it was.
```suggestion
        let containerClass = 'itemProgressBar';
        if (options && options.containerClass) {
            containerClass += ' ' + options.containerClass;
```